src/engine/polyline_compressor.cpp

```diff
@@ -61,24 +61,25 @@ std::vector<util::Coordinate> decodePolyline(const std::string &geometry_string)
 
     while (index < len)
     {
-        int b, shift = 0, result = 0;
-        do
+        int b = 0x20, shift = 0, result = 0;
```

Comment:
would it be possible to describe the logic behind this magical number in a comment around here?